boot/zephyr/CMakeLists.txt

```diff
@@ -54,6 +54,9 @@ macro(app_set_runner_args)
   endif()
 endmacro()
 
+# nRF Connect SDK application boilerplate, needed for out-of-tree Nordic boards.
```

Comment:
@SebastianBoe Is this line OK.  Can this interfere with boilerplate bellow?

Reply:
This is OK (and best-practice for NCS application's).